content/templates/implementations.md

```diff
@@ -0,0 +1,97 @@
+- <a href="https://www.figma.com/community/plugin/809860933081065308" target="_blank">Figma to Lottie</a> by @lottiefiles
+
+##### Apps
+
+- <a href="https://www.keyshapeapp.com/" target="_blank">Keyshape for Mac </a>
+- <a href="https://www.haikuforteams.com/animator/" target="_blank">Animator</a> by @haikuforteams
+- <a href="https://glaxnimate.mattbas.org/ " target="_blank">Glaxnimate </a> by @mbasaglia
+
+<br/>
+
+## Libraries
+
+<br/>
+
+##### Web
+
+- <a href="https://github.com/airbnb/lottie-web" target="_blank">lottie-web</a> - official
+- <a href="https://github.com/LottieFiles/lottie-player" target="_blank">lottie-player</a> - by @lottiefiles
+- <a href="https://github.com/LottieFiles/lottie-react" target="_blank">lottie-react</a> - by @lottiefiles
+- <a href="https://github.com/gamote/lottie-react" target="_blank">lottie-react</a> - by @gamote
+- <a href="https://github.com/LottieFiles/svelte-lottie-player" target="_blank">svelte-lottie-player</a> - by @lottiefiles
+- <a href="https://github.com/chenqingspring/vue-lottie" target="_blank">vue-lottie</a> - by @chenqingspring
+- <a href="https://github.com/chenqingspring/ng-lottie" target="_blank">ng-lottie</a> - Lottie for Angular by @chenqingspring
+- <a href="https://github.com/LottieFiles/dotlottie-web"  target="_blank">dotlottie-web</a>
+
+##### Android
+
+- <a href="https://github.com/airbnb/lottie-android" target="_blank">lottie-android</a> - official
+- <a href="https://github.com/airbnb/lottie-react-native" target="_blank">lottie-react-native</a> - official
+- <a href="https://github.com/martijn00/LottieXamarin" target="_blank">lottie-xamarin</a>
+- <a href="https://github.com/bradmartin/nativescript-lottie" target="_blank">native-script-lottie</a>
+- <a href="https://github.com/m1ga/ti.animation" target="_blank">ti.animation</a> - Appcelerator Titanium bindings
+- <a href="https://github.com/simolus3/fluttie" target="_blank">Fluttie</a> - Lottie for Flutter
+- <a href="https://github.com/LottieFiles/dotlottie-android" target="_blank">dotlottie-android</a>
+
+##### iOS
+
+- <a href="https://github.com/airbnb/lottie-ios" target="_blank">lottie-ios</a> - official
+- <a href="https://github.com/airbnb/lottie-react-native" target="_blank">lottie-react-native</a> - official
+- <a href="https://github.com/LottieFiles/dotlottie-ios" target="_blank">dotlottie-ios</a>
+
+##### Desktop
+
+- <a href="https://github.com/windows-toolkit/Lottie-Windows" target="_blank">lottie-windows</a>
+- <a href="https://github.com/azchohfi/LottieUWP" target="_blank">Lottie for UWP</a> - by @azchohfi
+- <a href="https://lottiefiles.com/desktop" target="_blank">LottieFiles Desktop</a>
+
+##### Multi-Platform
+
+- <a href="https://www.qt.io/blog/2019/03/08/announcing-qtlottie" target="_blank">QT</a>
+- <a href="https://skia.org/user/modules/skottie" target="_blank">Skottie</a> - Lottie Animation Player
```

Comment:
Link doesn't work, should be https://skia.org/docs/user/modules/skottie/ (missing docs part)